docs/exceptions/README.md

```diff
@@ -7,13 +7,61 @@ path_base_for_github_subdir:
 
 # Semantic Conventions for Exceptions
 
-**Status**: [Stable][DocumentStatus]
-
-This document defines semantic conventions for Exceptions.
+**Status**: [Stable][DocumentStatus], Unless otherwise specified.
 
 Semantic conventions for Exceptions are defined for the following signals:
 
 * [Exceptions on spans](exceptions-spans.md): Semantic Conventions for Exceptions associated with *spans*.
 * [Exceptions in logs](exceptions-logs.md): Semantic Conventions for Exceptions recorded in *logs*.
 
+## Reporting exceptions in instrumentation code
```

Comment:
Shouldn't this new section be part of the new document `recording-errors.md`? It seems more fitting there than here as part of the general readme.

Also then there's some duplicated information here and in the `recording-errors.md` such as what Span status to set, what to set for error.type and etc.

Reply:
good point, moved it